api/user.go

```diff
@@ -47,7 +47,11 @@ func GetUsers(c *gin.Context) {
 
 	var users []auth.User
 	db := config.DB()
-	err = db.Model(organization).Related(&users, "Users").Error
+	if id == 0 {
```

Comment:
You can write this: `err = db.Model(organization).Where(&auth.User{ID: uint(id)}).Related(&users, "Users").Error` instead of the if block. When userId is 0 the Where clause gets ignored by GORM.

Reply:
thx